includes/TaskManagers/PluginUninstallTaskManager.php

```diff
@@ -0,0 +1,145 @@
+<?php
+namespace NewfoldLabs\WP\Module\Onboarding\TaskManagers;
+
+use NewfoldLabs\WP\Module\Onboarding\Data\Options;
+use NewfoldLabs\WP\Module\Onboarding\Models\PriorityQueue;
+use NewfoldLabs\WP\Module\Onboarding\Tasks\PluginUninstallTask;
+
+/**
+ * Manages the execution of PluginUninstallTasks.
+ */
+class PluginUninstallTaskManager {
+
+	 /**
+	  * The number of times a PluginUninstallTask can be retried.
+	  *
+	  * @var int
+	  */
+	private static $retry_limit = 1;
+
+	private static $queue_name = 'plugin_uninstall_queue';
+
+	function __construct() {
+		// Ensure there is a thirty second option in the cron schedules
+		add_filter( 'cron_schedules', array( $this, 'add_thirty_seconds_schedule' ) );
+
+		// Thirty second cron hook
+		add_action( 'nfd_module_onboarding_plugin_uninstall_cron', array( $this, 'uninstall' ) );
+
+		// Register the cron task
+		if ( ! wp_next_scheduled( 'nfd_module_onboarding_plugin_uninstall_cron' ) ) {
+			wp_schedule_event( time(), 'thirty_seconds', 'nfd_module_onboarding_plugin_uninstall_cron' );
```

Comment:
Can uninstalls be faster than `30 seconds`?

Reply:
Yes, it can be but to be on the safer side I choose this as sometimes what if plugins are huge in size as in they save a lot of user data? That can take some time to wipe off the disk.